langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModel.java

```diff
@@ -407,16 +417,8 @@ private Response<AiMessage> generate(List<ChatMessage> messages,
                     logger.warn("Exception while calling model listener", e2);
                 }
             });
-            FinishReason exceptionFinishReason = contentFilterManagement(httpResponseException, "content_filter");
-            if (exceptionFinishReason != FinishReason.CONTENT_FILTER) {
-                throw httpResponseException;
-            }
 
-            return Response.from(
-                    aiMessage(httpResponseException.getMessage()),
-                    null,
-                    exceptionFinishReason
-            );
+            throw exception;
```

Comment:
All exceptions will be propagated to the caller as usual instead of returned to the user as a textual response from the LLM